scripts/__setup_taskwarrior_cron.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+SCRIPT_PATH="${HOME}/dev/dotfiles/scripts/__optimize_taskwarrior_db.sh"
+CRON_ENTRY="0 15 * * 0 ${SCRIPT_PATH} # Weekly Taskwarrior DB optimization (Sundays 3 PM)"
+
+if ! crontab -l 2>/dev/null | grep -q "__optimize_taskwarrior_db.sh"; then
+    (crontab -l 2>/dev/null; echo "$CRON_ENTRY") | crontab -
```

Comment:
Using `crontab -` replaces the entire crontab. Consider using a safer approach or adding more validation to ensure the crontab command succeeds and handle potential errors.